.github/workflows/test.yml

```diff
@@ -261,8 +155,26 @@ jobs:
       - name: Install Python packages
         uses: ./src/DDS-Router/.github/actions/install-python-packages
 
-      - name: Fetch Fast DDS repositories
-        uses: ./src/DDS-Router/.github/actions/fetch-fastdds-repos
+      - name: Get build eProsima dependencies Job Id
+        run: |
+          export JOB_ID=$(curl -sL $GITHUB_API_URL/repos/$GITHUB_REPOSITORY/actions/workflows/build_fastdds.yml/runs | \
+            jq '.workflow_runs[] | select(.status == "completed") | .id' | \
+            head -n 1)
+          echo "fastdds_job_id=${JOB_ID}" >> $GITHUB_ENV
+
+      - name: Download eProsima dependencies
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: build_fastdds.yml
+          path: /home/runner/work/fastdds/install
+          name: ubuntu_fastdds_install
+          run_id: ${{ env.fastdds_job_id }}
+
+      - name: Install eProsima dependencies
+        run: |
+          sudo cp -r /home/runner/work/fastdds/install/foonathan_memory_vendor/* /usr/local
```

Comment:
Are these copies really necessary? It seems like the source is being done in the original dir